backend/.env

```diff
@@ -1 +1,3 @@
 PUSHPIN_CONFIG_DIR=./pushpin_config
+DATABASE_NAME=tracim_test
+TMPFS_DIR=/var/lib/mysql
```

Comment:
Do we use the tmpfs trick outside of travis ? If not i'm not sure having this parameter here make sense.

Reply:
After rereading the code, it look like it's used also for local test, nice. Doesn't we profit of similar trick for sqlite/postgres ?

Reply:
For sqlite we'd need to put the test database file in a specific directory (as tmpfs applies to an entire directory). When I tested I didn't find a big difference with postgresql in Travis.